src/trace/span-inferrer.spec.ts

```diff
@@ -0,0 +1,240 @@
+import { SpanInferrer } from "./span-inferrer";
+import { SpanContext, TracerWrapper } from "./tracer-wrapper";
+const snssqsEvent = require("../../event_samples/snssqs.json");
+const snsEvent = require("../../event_samples/sns.json");
+const sqsEvent = require("../../event_samples/sqs.json");
+const ddbEvent = require("../../event_samples/dynamodb.json");
+const kinesisEvent = require("../../event_samples/kinesis.json");
+const eventBridgeEvent = require("../../event_samples/event-bridge.json");
```

Comment:
Nit: can we rename this to `eventbridge`, the `-` prevents it from showing up in search.